src/DependencyManagement/DependencyManager.cs

```diff
@@ -206,7 +206,12 @@ private void InstallSnapshotInForeground(PowerShell pwsh, ILogger logger)
         {
             try
             {
-                _installer.InstallSnapshot(_dependenciesFromManifest, _currentSnapshotPath, pwsh, logger);
+                _installer.InstallSnapshot(
+                    _dependenciesFromManifest,
+                    _currentSnapshotPath,
+                    pwsh,
+                    removeIfEquivalentToLatest: false,
```

Comment:
Could you please add a note why we set ```removeIfEquivalentToLatest: false```?

Reply:
Done.